examples/demoadc/DemoAdc.c

```diff
@@ -53,7 +53,7 @@ OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 #define PI 3.14159265358
 static int totSamples = 65536;
 static double frequency;
-static int pts;
+// static int pts;
```

Comment:
I would say delete the line; we have git history to remember

Reply:
me 2